build and installation creates unusable compiler

I must be missing something, but documentation really says very little to help me solve this issue.

```shell
user@debian:~/AdaptiveCpp/examples/build$ acpp
acpp [AdaptiveCpp compilation driver], Copyright (C) 2018-2023 Aksel Alpay and the AdaptiveCpp project
  AdaptiveCpp version: 24.02.0+git.3d794731.20240312.branch.develop
  Installation root: /usr/local
  Plugin LLVM version: 17, can accelerate CPU: True
  Available runtime backends:
     librt-backend-hip.so
     librt-backend-ocl.so
     librt-backend-omp.so
Usage: acpp <options>
```


```shell
user@debian:~/AdaptiveCpp/examples/a$ acpp example.cpp 
acpp warning: No optimization flag was given, optimizations are disabled by default. Performance may be degraded. Compile with e.g. -O2/-O3 to enable optimizations.
example.cpp:1:10: fatal error: 'cassert' file not found
    1 | #include <cassert>
      |          ^~~~~~~~~
1 error generated.
```


```shell
user@debian:~/AdaptiveCpp/examples/build$ make VERBOSE=1
/usr/bin/cmake -S/home/user/AdaptiveCpp/examples -B/home/user/AdaptiveCpp/examples/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/user/AdaptiveCpp/examples/build/CMakeFiles /home/user/AdaptiveCpp/examples/build//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/user/AdaptiveCpp/examples/build'
make  -f bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/build.make bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/depend
make[2]: Entering directory '/home/user/AdaptiveCpp/examples/build'
cd /home/user/AdaptiveCpp/examples/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/user/AdaptiveCpp/examples /home/user/AdaptiveCpp/examples/bruteforce_nbody /home/user/AdaptiveCpp/examples/build /home/user/AdaptiveCpp/examples/build/bruteforce_nbody /home/user/AdaptiveCpp/examples/build/bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/user/AdaptiveCpp/examples/build'
make  -f bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/build.make bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/build
make[2]: Entering directory '/home/user/AdaptiveCpp/examples/build'
[ 50%] Building CXX object bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/bruteforce_nbody.cpp.o
cd /home/user/AdaptiveCpp/examples/build/bruteforce_nbody &&  /usr/local/lib/cmake/AdaptiveCpp/syclcc-launcher --launcher-cxx-compiler=/usr/bin/c++ --launcher-syclcc=/usr/local/bin/acpp  /usr/bin/c++ -DHIPSYCL_DEBUG_LEVEL=2 -I/home/user/AdaptiveCpp/examples/build -I/home/user/AdaptiveCpp/examples -isystem /usr/local/include/AdaptiveCpp -O3 -DNDEBUG -std=c++17 -MD -MT bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/bruteforce_nbody.cpp.o -MF CMakeFiles/bruteforce_nbody.dir/bruteforce_nbody.cpp.o.d -o CMakeFiles/bruteforce_nbody.dir/bruteforce_nbody.cpp.o -c /home/user/AdaptiveCpp/examples/bruteforce_nbody/bruteforce_nbody.cpp
/home/user/AdaptiveCpp/examples/bruteforce_nbody/bruteforce_nbody.cpp:28:10: fatal error: 'iostream' file not found
   28 | #include <iostream>
      |          ^~~~~~~~~~
1 error generated.
make[2]: *** [bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/build.make:78: bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/bruteforce_nbody.cpp.o] Error 1
make[2]: Leaving directory '/home/user/AdaptiveCpp/examples/build'
make[1]: *** [CMakeFiles/Makefile2:98: bruteforce_nbody/CMakeFiles/bruteforce_nbody.dir/all] Error 2
make[1]: Leaving directory '/home/user/AdaptiveCpp/examples/build'
make: *** [Makefile:136: all] Error 2
user@debian:~/AdaptiveCpp/examples/build$ 
```





Initial cmake output:

```
user@debian:~/AdaptiveCpp/build$ cmake ..
On branch develop
Your branch is up to date with 'origin/develop'.

nothing to commit, working tree clean
-- The C compiler identification is GNU 13.2.0
-- The CXX compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.83.0/BoostConfig.cmake (found version "1.83.0") found components: context fiber 
-- hip::amdhip64 is SHARED_LIBRARY
-- /usr/bin/c++: CLANGRT compiler options not supported.
-- Performing Test HAVE_FFI_CALL
-- Performing Test HAVE_FFI_CALL - Success
-- Found FFI: /usr/lib/x86_64-linux-gnu/libffi.so  
-- Could NOT find LibEdit (missing: LibEdit_INCLUDE_DIRS LibEdit_LIBRARIES) 
-- Performing Test Terminfo_LINKABLE
-- Performing Test Terminfo_LINKABLE - Success
-- Found Terminfo: /usr/lib/x86_64-linux-gnu/libtinfo.so  
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.3")  
-- Found zstd: /usr/lib/x86_64-linux-gnu/libzstd.so  
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.14") 
-- Found CURL: /usr/lib/x86_64-linux-gnu/libcurl.so (found version "8.6.0")  
-- Building hipSYCL against LLVM configured from /usr/lib/llvm-17/cmake
-- Selecting clang: /usr/lib/llvm-17/bin/clang++
-- Using clang include directory: /usr/lib/llvm-17/lib/clang/17/include/..
-- Looking for C++ include filesystem
-- Looking for C++ include filesystem - found
-- Performing Test CXX_FILESYSTEM_NO_LINK_NEEDED
-- Performing Test CXX_FILESYSTEM_NO_LINK_NEEDED - Success
patching file /home/user/AdaptiveCpp/build/_deps/llvmspirvtranslator-src/lib/SPIRV/SPIRVInternal.h
Hunk #1 succeeded at 181 (offset 1 line).
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.8.1") 
-- SPIR-V Headers location is not specified. Will try to download
          spirv.hpp from https://github.com/KhronosGroup/SPIRV-Headers into
          /home/user/AdaptiveCpp/build/_deps/llvmspirvtranslator-build/SPIRV-Headers
-- The CXX compiler identification is GNU 13.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The C compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test HAVE_FFI_CALL
-- Performing Test HAVE_FFI_CALL - Success
-- Found FFI: /usr/lib/x86_64-linux-gnu/libffi.so  
-- Could NOT find LibEdit (missing: LibEdit_INCLUDE_DIRS LibEdit_LIBRARIES) 
-- Performing Test Terminfo_LINKABLE
-- Performing Test Terminfo_LINKABLE - Success
-- Found Terminfo: /usr/lib/x86_64-linux-gnu/libtinfo.so  
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.3")  
-- Found zstd: /usr/lib/x86_64-linux-gnu/libzstd.so  
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.14") 
-- Found CURL: /usr/lib/x86_64-linux-gnu/libcurl.so (found version "8.6.0")  
-- Linker detection: GNU ld
-- Performing Test C_SUPPORTS_FPIC
-- Performing Test C_SUPPORTS_FPIC - Success
-- Performing Test CXX_SUPPORTS_FPIC
-- Performing Test CXX_SUPPORTS_FPIC - Success
-- Building with -fPIC
-- Performing Test C_SUPPORTS_FNO_SEMANTIC_INTERPOSITION
-- Performing Test C_SUPPORTS_FNO_SEMANTIC_INTERPOSITION - Success
-- Performing Test CXX_SUPPORTS_FNO_SEMANTIC_INTERPOSITION
-- Performing Test CXX_SUPPORTS_FNO_SEMANTIC_INTERPOSITION - Success
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG - Success
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test C_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW
-- Performing Test C_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW - Failed
-- Performing Test CXX_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW
-- Performing Test CXX_SUPPORTS_WERROR_UNGUARDED_AVAILABILITY_NEW - Failed
-- Performing Test C_SUPPORTS_CMAKE_CXX_FLAGS
-- Performing Test C_SUPPORTS_CMAKE_CXX_FLAGS - Success
-- Performing Test CXX_SUPPORTS_CMAKE_CXX_FLAGS
-- Performing Test CXX_SUPPORTS_CMAKE_CXX_FLAGS - Success
-- Performing Test CXX_SUPPORTS_MISSING_FIELD_INITIALIZERS_FLAG
-- Performing Test CXX_SUPPORTS_MISSING_FIELD_INITIALIZERS_FLAG - Success
-- Performing Test C_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG
-- Performing Test C_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG - Success
-- Performing Test CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG
-- Performing Test CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG - Success
-- Performing Test C_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG
-- Performing Test C_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG - Failed
-- Performing Test CXX_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG
-- Performing Test CXX_SUPPORTS_COVERED_SWITCH_DEFAULT_FLAG - Failed
-- Performing Test CXX_SUPPORTS_CLASS_MEMACCESS_FLAG
-- Performing Test CXX_SUPPORTS_CLASS_MEMACCESS_FLAG - Success
-- Performing Test CXX_SUPPORTS_REDUNDANT_MOVE_FLAG
-- Performing Test CXX_SUPPORTS_REDUNDANT_MOVE_FLAG - Success
-- Performing Test CXX_SUPPORTS_PESSIMIZING_MOVE_FLAG
-- Performing Test CXX_SUPPORTS_PESSIMIZING_MOVE_FLAG - Success
-- Performing Test CXX_SUPPORTS_NOEXCEPT_TYPE_FLAG
-- Performing Test CXX_SUPPORTS_NOEXCEPT_TYPE_FLAG - Success
-- Performing Test CXX_WONT_WARN_ON_FINAL_NONVIRTUALDTOR
-- Performing Test CXX_WONT_WARN_ON_FINAL_NONVIRTUALDTOR - Failed
-- Performing Test CXX_SUPPORTS_SUGGEST_OVERRIDE_FLAG
-- Performing Test CXX_SUPPORTS_SUGGEST_OVERRIDE_FLAG - Success
-- Performing Test CXX_WSUGGEST_OVERRIDE_ALLOWS_ONLY_FINAL
-- Performing Test CXX_WSUGGEST_OVERRIDE_ALLOWS_ONLY_FINAL - Success
-- Performing Test C_WCOMMENT_ALLOWS_LINE_WRAP
-- Performing Test C_WCOMMENT_ALLOWS_LINE_WRAP - Failed
-- Performing Test C_SUPPORTS_STRING_CONVERSION_FLAG
-- Performing Test C_SUPPORTS_STRING_CONVERSION_FLAG - Failed
-- Performing Test CXX_SUPPORTS_STRING_CONVERSION_FLAG
-- Performing Test CXX_SUPPORTS_STRING_CONVERSION_FLAG - Failed
-- Performing Test CXX_SUPPORTS_MISLEADING_INDENTATION_FLAG
-- Performing Test CXX_SUPPORTS_MISLEADING_INDENTATION_FLAG - Success
-- Performing Test C_SUPPORTS_CTAD_MAYBE_UNSPPORTED_FLAG
-- Performing Test C_SUPPORTS_CTAD_MAYBE_UNSPPORTED_FLAG - Failed
-- Performing Test CXX_SUPPORTS_CTAD_MAYBE_UNSPPORTED_FLAG
-- Performing Test CXX_SUPPORTS_CTAD_MAYBE_UNSPPORTED_FLAG - Success
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FDATA_SECTIONS
-- Performing Test C_SUPPORTS_FDATA_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS - Success
-- Looking for os_signpost_interval_begin
-- Looking for os_signpost_interval_begin - not found
-- Found LLVM: 17.0.6
-- Checking for one of the modules 'SPIRV-Tools'
-- Configuring done (5.4s)
-- Generating done (0.0s)
-- Build files have been written to: /home/user/AdaptiveCpp/build/_deps/llvmspirvtranslator-build
-- Performing Test HAS_MCPU_NATIVE
-- Performing Test HAS_MCPU_NATIVE - Success
-- Performing Test HAS_MARCH_NATIVE
-- Performing Test HAS_MARCH_NATIVE - Success
-- Found Threads: TRUE  
CMake Deprecation Warning at build/_deps/ocl-cxx-headers-src/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
CMake Deprecation Warning at build/_deps/ocl-cxx-headers-src/examples/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Default compilation target(s): generic
-- Configuring done (16.5s)
-- Generating done (0.1s)
-- Build files have been written to: /home/user/AdaptiveCpp/build
```


For completeness, Debian testing, amd64.

hipcc 5.7.1

gcc is happy, and I do have things:

```shell
$ dpkg -l | grep libstdc++
ii  libstdc++-13-dev:amd64                                      13.2.0-18                                 amd64        GNU Standard C++ Library v3 (development files)
ii  libstdc++-13-dev-arm64-cross                                13.2.0-12cross1                           all          GNU Standard C++ Library v3 (development files) (arm64)
ii  libstdc++-13-dev-armhf-cross                                13.2.0-12cross1                           all          GNU Standard C++ Library v3 (development files) (armhf)
ii  libstdc++-13-dev-i386-cross                                 13.2.0-12cross1                           all          GNU Standard C++ Library v3 (development files) (i386)
ii  libstdc++-arm-none-eabi-dev                                 15:13.2.rel1-2+26                         all          GNU Standard C++ Library v3 for ARM Cortex-R/M processors (headers)
ii  libstdc++-arm-none-eabi-newlib                              15:13.2.rel1-2+26                         all          GNU Standard C++ Library v3 for ARM Cortex-R/M processors (newlib)
ii  libstdc++6:amd64                                            14-20240201-3                             amd64        GNU Standard C++ Library v3
ii  libstdc++6:i386                                             14-20240201-3                             i386         GNU Standard C++ Library v3
ii  libstdc++6-arm64-cross                                      14-20240127-1cross1                       all          GNU Standard C++ Library v3 (arm64)
ii  libstdc++6-armhf-cross                                      14-20240127-1cross1                       all          GNU Standard C++ Library v3 (armhf)
ii  libstdc++6-dbgsym:amd64                                     14-20240201-3                             amd64        debug symbols for libstdc++6
ii  libstdc++6-dbgsym:i386                                      14-20240201-3                             i386         debug symbols for libstdc++6
ii  libstdc++6-i386-cross                                       14-20240127-1cross1                       all          GNU Standard C++ Library v3 (i386)
```

and

```shell
# dpkg -l | grep clang
ii  clang-17                                                    1:17.0.6-9                                amd64        C, C++ and Objective-C compiler
ii  clang-format-17                                             1:17.0.6-9                                amd64        Tool to format C/C++/Obj-C code
ii  clang-tools-17                                              1:17.0.6-9                                amd64        clang-based tools for C/C++ developments
ii  libclang-17-dev                                             1:17.0.6-9                                amd64        Clang library - Development package
ii  libclang-common-17-dev:amd64                                1:17.0.6-9                                amd64        Clang library - Common development package
ii  libclang-cpp17-dev                                          1:17.0.6-9                                amd64        C++ interface to the Clang library
ii  libclang-cpp17t64                                           1:17.0.6-9                                amd64        C++ interface to the Clang library
ii  libclang-rt-17-dev:amd64                                    1:17.0.6-9                                amd64        Compiler-rt - development package
ii  libclang1-17t64                                             1:17.0.6-9                                amd64        C interface to the Clang library
ii  libopencl-clang-17-dev:amd64                                17.0.0-2.1                                amd64        thin wrapper for clang -- development files
ii  libopencl-clang17t64:amd64                                  17.0.0-2.1                                amd64        thin wrapper for clang
root@debian:~# 
```



But you are right, it is clang issue:

```
$ cat hello.cc 
#include <iostream>

int main() {
  std::cout << "Hello\n";
  return 0;
}
$ clang-17 hello.cc 
hello.cc:1:10: fatal error: 'iostream' file not found
    1 | #include <iostream>
      |          ^~~~~~~~~~
1 error generated.
```

```
root@debian:~# clang++-17 -v /dev/null
Debian clang version 17.0.6 (9)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
Found candidate GCC installation: /usr/bin/../lib/gcc-cross/i686-linux-gnu/13
Found candidate GCC installation: /usr/bin/../lib/gcc/x86_64-linux-gnu/13
Found candidate GCC installation: /usr/bin/../lib/gcc/x86_64-linux-gnu/14
Selected GCC installation: /usr/bin/../lib/gcc/x86_64-linux-gnu/14
Candidate multilib: .;@m64
Selected multilib: .;@m64
Found HIP installation: /usr, version 5.7.31921
 "/usr/bin/ld" -pie --hash-style=gnu --build-id --eh-frame-hdr -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o a.out /lib/x86_64-linux-gnu/Scrt1.o /lib/x86_64-linux-gnu/crti.o /usr/bin/../lib/gcc/x86_64-linux-gnu/14/crtbeginS.o -L/usr/bin/../lib/gcc/x86_64-linux-gnu/14 -L/usr/bin/../lib/gcc/x86_64-linux-gnu/14/../../../../lib64 -L/lib/x86_64-linux-gnu -L/lib/../lib64 -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib64 -L/lib -L/usr/lib /dev/null -lstdc++ -lm -lgcc_s -lgcc -lc -lgcc_s -lgcc /usr/bin/../lib/gcc/x86_64-linux-gnu/14/crtendS.o /lib/x86_64-linux-gnu/crtn.o
/usr/bin/ld: cannot find -lstdc++: No such file or directory
clang++-17: error: linker command failed with exit code 1 (use -v to see invocation)
root@debian:~# 
```



Hmm. I think it is trying to use libstdc++ 14, for some reasons.


(gcc / g++ of course do work fine)

--- a/doc/install-llvm.md
+++ b/doc/install-llvm.md
@@ -84,3 +84,31 @@ If AdaptiveCpp does not select the right clang++ or include directories, use the
 
 * `-DCLANG_EXECUTABLE_PATH=/path/to/clang++` must be pointed to the `clang++` executable from this LLVM installation.
 * `-DCLANG_INCLUDE_PATH=/path/to/clang-includes` must be pointed to the clang internal header directory. Typically, this is something like `$LLVM_INSTALL_PREFIX/include/clang/<llvm-version>/include`. Newer ROCm versions will require the parent directory instead, i.e. `$LLVM_INSTALL_PREFIX/include/clang/<llvm-version>`. This is only important for the ROCm backend.
+
+## If clang/LLVM does not find C++ standard library headers
+
+AdaptiveCpp relies on a working clang installation. clang in turn does not come with its own C++ standard library, but instead assumes that it can use the one from the system.
+
+### Missing packages
+
+If it does not find C++ standard library headers, it might be because you are missing a `libstdc++-dev` package or similar.
+
+If you have multiple gcc versions installed on your system, it is also possible that clang wants a specific libstdc++ version to match a gcc version that it has selected. You can use `clang++ -v /dev/null` to find out which gcc and libstdc++ version clang wants to use.
+*Note: This command is expected to fail with a linker error. It should print something like "Selected GCC installation: ..." which is the information you are looking for.*
+
+### GCC toolchain/C++ standard library is in a non-standard location
+
+If the C++ standard library is in a non-standard location (this might typically be the case on a cluster), it is also possible that clang does not find it.
+
+In that case, clang can be informed about the location of the GCC toolchain using the `--gcc-toolchain=...` flag.
+*Note:* This flag is very picky about the directory!. It needs to be provided with the parent directory that contains the `lib/gcc/<arch>/<version>` subdirectories. On a Cray system, this might e.g. be `--gcc-toolchain=/opt/cray/pe/gcc/12.2.0/snos`. If an incorrect path is provided, clang may silently ignore the argument. Use `clang++ -v /dev/null` as described above to check that it has accepted the GCC installation.
+
+While in theory AdaptiveCpp can be made to use that flag when compiling, a simpler solution to make that change permanent is to use clang configuration files ([details in the clang documentation](https://clang.llvm.org/docs/UsersManual.html#id25)):
+1. In the directory where `clang++` lives (e.g. some `bin` directory), create a new file `<name>.cfg`, replacing `<name>` with something of your choice.
+2. Put any flags that you want clang to use by default into this file, such as the `--gcc-toolchain=...` flag.
+3. Create a symlink to clang in the same directory that `clang++` and the config file exist in: `ln -s clang++ <name>-clang++`, again replacing `<name>` with the name of your config file.
+4. When invoking clang with the symlink, clang will automatically apply the configuration file. Now use `<name>-clang++` in the build process of AdaptiveCpp instead of clang++, e.g. use the cmake flags `-DCMAKE_CXX_COMPILER=<name>-clang++ -DCLANG_EXECUTABLE_PATH=/path/to/<name>-clang++`.
+
+
+
+

